release/models/policy/openconfig-routing-policy.yang

```diff
@@ -51,11 +51,13 @@ module openconfig-routing-policy {
     Evaluation of each policy definition proceeds by evaluating its
     corresponding individual policy statements in order.  When a
     condition statement in a policy statement is satisfied, the
-    corresponding action statement is executed.  If the action
-    statement has a final disposition configured as policy result, either
-    accept-route or reject-route, evaluation of the current policy definition
-    stops, and no further policy statements are evaluated. In case there is a
-    policy chain, no further policy definitions in the chain are evaluated.
+    corresponding action statement is executed.  A policy statement with no
+    conditions defined is evaluated as statisfied, meaning all routes are
+    matched.  If the action statement has a final disposition configured as
+    policy result, either accept-route or reject-route, evaluation of the
+    current policy definition stops, and no further policy statements are
+    evaluated. In case there is a policy chain, no further policy definitions
+    in the chain are evaluated.
```

Comment:
Interested in @xuqma's view here -- is it painful to add explicit "match everything" terms per my suggestion above?

Reply:
The proposed description change reads for policy-forwarding: `If no rules are specified, then the policy will match all packets.`

And for routing-policy:
`A policy statement with no conditions defined is evaluated as statisfied, meaning all routes are matched.`